notebook/static/tree/js/main.js

```diff
@@ -115,6 +115,18 @@ requirejs([
         )
     );
 
+    $("#refresh_notebook_list").click(function(){
+        if($(this).attr("aria-label")=="pressing refresh button"){
+            $(this).attr("aria-label","pressing refresh button.");
+        }
+        else if($(this).attr("aria-label")=="pressing refresh button."){
+            $(this).attr("aria-label","pressing refresh button");
```

Comment:
What's this for? It seems to toggle the "." at the end, but I don't understand the aim of that.

It should be integrated with the translation, and I don't like the check against a human-readable string.